pkgs/development/python-modules/crytic-compile/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, python3Packages }:
```

Comment:
for python-modules, you import individual packages
```suggestion
{ stdenv, buildPythonPackage, fetchPypi, pysha3 }:
```

Reply:
Done